src/main/webapp/app/shared/circular-progress-bar/circular-progress-bar.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="progress-circle" [attr.data-percentage]="progressInPercent">
+    <span class="progress-left">
+        <span class="progress-bar-circle" [style.border-color]="circleColor"></span>
```

Comment:
So if I understand this correctly, Angular will call the `circleColor` function to get the color quite often (change detection?), even without `progressInPercent` changing. If so, wouldn't it make sense to cache that value in a field instead?

Reply:
Good catch: 4f1d3097574eb0d550b45758427f47fce23b2d0d